src/components/MenuLeft/MenuLeft.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './MenuLeft.scss';
+import smilies from '../../assets/images/smilies.png';
+import menuIcon from '../../assets/icons/menu-icon.png';
+import { FormattedMessage } from 'react-intl';
+
+function MenuLeft() {
+  return (
```

Comment:
You can do: 

```suggestion
const MenuLeft = ()  => (
```
and also avoid the return sentence 😄